tests/onebox.go

```diff
@@ -998,3 +1030,20 @@ func sdkClientFactoryProvider(
 		dynamicconfig.WorkerStickyCacheSize.Get(dc),
 	)
 }
+
+func (c *temporalImpl) overrideDynamicConfigByKey(t *testing.T, name dynamicconfig.Key, value any) {
+	constrainedValues := c.dcClient.GetValue(name)
+	existed := len(constrainedValues) == 0
+	c.dcClient.OverrideValueByKey(name, value)
+	t.Cleanup(func() {
+		if existed {
+			var values []any
+			for _, v := range constrainedValues {
+				values = append(values, v)
+			}
+			c.dcClient.OverrideValueByKey(name, values)
```

Comment:
doesn't this not work without the type check in GetValue? it'll re-wrap the list...
but also, why copy it to a `[]any` at all?

Reply:
Yeah, that was a mistake.

Reply:
Done